.circleci/config.yml

```diff
@@ -258,32 +258,29 @@ jobs:
       - checkout
       - attach_workspace:
           at: ~/
-      - run:
-          name: "Install cloudfoundry dependencies"
-          command: |
-            source ./scripts/pipeline/deb-basic-deps.sh
-            source ./scripts/pipeline/deb-cf-install.sh
+      # - run:
+          # name: "Install cloudfoundry dependencies"
+          # command: |
+          #   source ./scripts/pipeline/deb-basic-deps.sh
+          #   source ./scripts/pipeline/deb-cf-install.sh
```

Comment:
If you do not need something any longer, please remove it instead of only commenting it out.

Reply:
i kept it cause i didnt know if we needed it for later on but can always put it back